sklearn/utils/estimator_checks.py

```diff
@@ -1730,8 +1730,8 @@ def check_decision_proba_consistency(name, Estimator):
     # predict_proba methods has outputs with perfect rank correlation.
 
     rnd = np.random.RandomState(0)
-    X_train = (2*rnd.uniform(size=(10, 4))).astype(int)
-    y = X_train[:, 0]
+    X_train = np.random.randint(2, size=(10, 4))
```

Comment:
while there might be some benefit in using np.random directly, we usually use determinism (i.e. `rnd`) to avoid heisenbugs.